docs/docs/model-configuration.mdx

```diff
@@ -9,6 +9,17 @@ abstract: The configuration file defines the components and policies that your m
 The recipe key allows for different types of config and model architecture.
 Currently, "default.v1" and the experimental "graph.v1" recipes are supported.
 
+:::info New in 3.5
+
+The config file now includes a new mandatory key `assistant_id` which represents the unique assistant identifier.
+
+:::
+
+The `assistant_id` key must specify a unique value to distinguish multiple assistants in deployment.
+The assistant identifier will be propagated to each event's metadata, alongside the model id.
+Note that if the config file does not include this required key or the placeholder default value is not replaced, a random
+assistant name will be generated and added to the configuration everytime when running `rasa train`.
```

Comment:
I'm wondering why there is a difference in behaviour between `rasa interactive`, `rasa data validate` and `rasa train`. The first 2 commands seem to be erroring but the last one works as expected with a sensible default. Can't we have this default behaviour for all the commands?

Reply:
`rasa interactive` has 2 paths: training a model from scratch or running interactive learning with a pre-trained model). It's in the last path (of using a pretrained model) that the exiting with an error behaviour is implemented, because of the missing parameter in the model metadata. In the first path, code that is used by `rasa train` is re-used and therefore that works like `rasa train`.

Reply:
sounds good, thanks for the explanation (it was right before my eyes 🤦🏻 )